### PR TITLE
Fix usage instructions

### LINKED ISSUE #564: Usage section of readme.md is incorrect
1. If you haven't built promu before, it must be built. And you don't have promu in PATH, so it won't be found.
2. The . should be removed according to #458 (and promu related issue).

### PR DESCRIPTION
**Pull request by eloekset**
Fixes #564 
- First promu must be built
- `promu build -v` is the correct command to build **windows_exporter** (no . as argument for local directory)

### CODE REVIEW

**Comment by carlpett** [comment]
Hi @eloekset,
Thanks! In a typical Go setup, `go get ...` should actually automatically build the target and put the binary in your PATH. Might you be missing `GOPATH\bin` on your machine?

**Comment by eloekset** [comment]
Ok, let me try again on another Windows machine. It's possible that I built it manually without checking if promu.exe was built automatically.

I can fix the PR if I verify the above, so that the only change is removal of the . for local folder (which it isn't).

**Comment by eloekset** [comment]
It's updated now, @carlpett.

I've verified that it works exactly like the instructions say on a Windows environment where Go has just been installed and **promu** and **windows_exporter** have just been downloaded using `go get -u`.

```
PS C:\Users\eivin\go\src> go get -u github.com/prometheus-community/windows_exporter
PS C:\Users\eivin\go\src> cd $env:GOPATH\src\github.com\prometheus-community\windows_exporter
PS C:\Users\eivin\go\src\github.com\prometheus-community\windows_exporter> promu build -v
Using config file: .promu.yml
Running command: build [-v]
Building binary: windows_exporter
 >   windows_exporter.exe
Building binary: go build -o windows_exporter.exe -ldflags -X github.com/prometheus/common/version.Version=0.13.0-18-ga03f9ef -X github.com/prometheus/common/version.Revision=a03f9ef01bd58a6dda070fe00ffba8d441fc9245 -X github.com/prometheus/common/version.Branch=master -X github.com/prometheus/common/version.BuildUser=yoga-eivind\eivin@Yoga-Eivind -X github.com/prometheus/common/version.BuildDate=20200706-08:31:00  -extldflags '-static' github.com/prometheus-community/windows_exporter
 +  go build -o windows_exporter.exe -ldflags -X github.com/prometheus/common/version.Version=0.13.0-18-ga03f9ef -X github.com/prometheus/common/version.Revision=a03f9ef01bd58a6dda070fe00ffba8d441fc9245 -X github.com/prometheus/common/version.Branch=master -X github.com/prometheus/common/version.BuildUser=yoga-eivind\eivin@Yoga-Eivind -X github.com/prometheus/common/version.BuildDate=20200706-08:31:00  -extldflags '-static' github.com/prometheus-community/windows_exporter
```
